NetCoreMentoring/src/NetCoreMentoring.Core/Models/Category.cs

```diff
@@ -7,5 +7,7 @@ public class Category
         public string CategoryName { get; set; }
 
         public string Description { get; set; }
+
+        public byte[] Picture { get; set; }
```

Comment:
Try to use clearer naming: `ImageBytes`?

Reply:
done